osu.Game/Beatmaps/Drawables/UpdateableBeatmapBackgroundSprite.cs

```diff
@@ -45,24 +45,7 @@ protected override DelayedLoadWrapper CreateDelayedLoadWrapper(Drawable content,
 
         protected override Drawable CreateDrawable(BeatmapInfo model)
         {
-            Drawable drawable;
-
-            var localBeatmap = beatmaps.GetWorkingBeatmap(model);
```

Comment:
I'm pretty sure we do still want this, e.g. `MultiplayerBackgroundSprite` sets the beatmap from the `Room` without doing a local lookup itself.

Reply:
i didn't remove this

Reply:
Oh my bad I misread this as QueryBeatmap...